### PR TITLE
Initial abs build

### PR DESCRIPTION
**Pull request by mcg1969**
Needed to create a new branch and PR to solve some Prefect issues

Initial anaconda-recipes build, no apparent changes needed

Actions:
- add several fields in the `about` section

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
There isn't the linux-64 builder machine on Prefect. We have to reopen the PR

**Comment by skupr-anaconda** [comment]
Should we add flags for CMake in build.sh and bld.bat: 
`cmake -DENABLE_DATE_TESTING=ON -DBUILD_TZ_LIB=ON ../`
see a comment about CMake https://github.com/HowardHinnant/date/tree/v3.0.1#build--test

And add multithreading.

**Comment by mcg1969** [comment]
Interesting idea, I will give it a shot

**Comment by mcg1969** [comment]
@skupr-anaconda I believe I've got it. Can you please take a look and see if you agree?

Note that I had to add `libcurl` as a build, and run dependency, as well as a run_exports dependency. I did that because the CMake output has this:
```
-- Found CURL: $PREFIX/lib/libcurl.dylib (found version "7.82.0")  
```
Without the bulid dependency it was picking up system libcurl. I'm concerned that it might be leaving symbols in `libdate-tz.a` that require libcurl to resolve.

**Comment by mcg1969** [comment]
Looks like Windows still isn't quite right. I'll work on that

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,17 +9,24 @@ package:
...
 build:
   number: 0
+  run_exports:
+    strong:
+      - libcurl
 
 requirements:
-  build:
+  host:
     - {{ compiler('c') }}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:21]: What if the sections will be:
```
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - cmake
    - ninja
    - libcurl
  host:
    - libcurl 
```

Comment (mcg1969) [reply recipe/meta.yaml:21]: Is that right? I always thought that the compilers and build tools need to be in the `host` section.

Comment (skupr-anaconda) [reply recipe/meta.yaml:21]: No, compilers need to be in `req/build`

Comment (mcg1969) [reply recipe/meta.yaml:21]: got it, thanks! I just got mixed up in my brain

Comment (mcg1969) [reply recipe/meta.yaml:21]: @skupr-anaconda ok, that's much better! Thank you for the help!

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**